.github/workflows/ci.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ${{ env.SCIE_PANTS_DEV_CACHE }}
-          key: ${{ matrix.os }}-scie-pants-v5
+          key: ${{ matrix.os }}-scie-pants-v6
```

Comment:
What's the v represent? Not the scie-pants version, surely

Reply:
The cache version, aka a manual cache dump knob just as being used here.